addons/point_of_sale/static/src/app/main.js

```diff
@@ -60,10 +60,10 @@ whenReady(() => {
 function registerServiceWorker() {
     // Register the service worker for the POS
     const urlsToCache = JSON.parse(odoo.urls_to_cache);
-    const translationsHash = session.cache_hashes?.translations;
     const lang = jsToPyLocale(user.lang || document.documentElement.getAttribute("lang"));
     const translationURL = session.translationURL || "/web/webclient/translations";
-    const url = `${translationURL}/${translationsHash}?lang=${lang}`;
+    const unique = new Date().getTime().toString();
+    const url = `${translationURL}/${unique}?lang=${lang}`;
```

Comment:
shouldn't the fetch of translations be removed from the urlsToCache thing? It's a bit counter intuitive as from one side, we have a unique url that will never do a cache hit, and from the other, we're caching the request